.gitignore

```diff
@@ -5,3 +5,5 @@ out/
 .idea/
 
 *.semanticdb
+.bsp
+.scala-build
```

Comment:
The `.scala-build` shouldn't be needed here. Ideally, no `.scala-build` directory should be created at the root (this is checked in the tests, to ensure `.scala-build` is created where expected, rather than naively in `os.pwd`). Better put that line in a `.gitignore` in a sub-directory.